### DISCUSSION TITLE
Row count in View Data?

### DISCUSSION THREAD

**Discussion by RedTailedHawk** [dbeaver/dbeaver#9528] [Q&A] [answered]
Is there any way to see the row count for a table in the View Data screen?  Looks like the default number of sample rows is 200, but would be helpful to see the total count(*) somewhere on that view.

Thanks.

**Comment by kseniiaguzeeva** [comment, marked as answer]
You can use the btn Calculate total row count in the bottom toolbar.
![image](https://user-images.githubusercontent.com/51405061/90231587-16be9880-de24-11ea-9c4b-18eec2a37941.png)
For more options please see https://github.com/dbeaver/dbeaver/wiki/Data-Editor.
If you need, it is possible to change fetch size (200 rows):
1 Change row number manualy in Bottom toolbar and press Refresh
![dbeaver_c5sBZ6vWXn](https://user-images.githubusercontent.com/51405061/90231989-a8c6a100-de24-11ea-88e4-f95738e65c4d.png)
2 Press Settings btn in Bottom toolbar and change ResultSet fetch size
![dbeaver_1NvsuMuO0o](https://user-images.githubusercontent.com/51405061/90231978-a3695680-de24-11ea-8cb1-b7ab67fd31bc.png)
